Makefile

```diff
@@ -130,15 +131,11 @@ clean-proto-defs: | $(clean)
 goimports:
```

Comment:
Do you want to do the same thing you did for govendor to goimports?

Reply:
Good idea - done.